lib/Config/Menu.php

```diff
@@ -700,6 +706,7 @@ function newletterEditor() {
     $data = array(
       'shortcodes' => ShortcodesHelper::getShortcodes(),
       'settings' => $this->settings->getAll(),
+      'user_flags' => $this->user_flags->getAll(),
```

Comment:
Do we really need all flags in the template? Would it be better to explicitly pass only those we really need?